powerpacks.go

```diff
@@ -33,6 +33,7 @@ type NumberPoolResponse struct {
 	Number                        string `json:"number,omitempty"`
 	Type                          string `json:"Type,omitempty"`
 	Country_iso2                  string `json:"country_iso2,omitempty"`
+	Service                       string `json:"service,omitempty"`
```

Comment:
Since this will be permanent, please check with PM if `Service` is the term we will be using to differentiate between different messaging "channels". Here are some alternatives:

* Channel
* Platform

Which one is picked, it doesn't matter much. However, once picked, we will have to be consistent with its usage through-out all SDKs and API documentation.

Reply:
Please ignore this. Will confirm with PM before release.